trajectory_publisher_saver_node:
  namespace: ""
  odom_frame: "odom"
  odom_topic: "odom"
  marker_topic: "trajectory_markers"
  marker_interval: 0.1
  position_tolerance: 0.02
  yaw_tolerance: 0.02


trajectory_reader_publisher_node:
  odom_frame: "odom"
  marker_topic: "trajectory_markers"